### ISSUE TITLE
"Waiting for other folder" language is confusing

### ISSUE THREAD

**Issue by jerrac** [owncloud/client#4022]
### Expected behaviour

When client is waiting for another account to sync, it should say so.
### Actual behaviour

Client says that it's waiting for another "folder". 
### Steps to reproduce
1. Set up multiple owncloud accounts in the client.
2. Add a ton of data to one so that it takes a long time to sync.
3. Look at the status of the other account.
### Server configuration

Operating system: Ubuntu 14.04

Web server: Apache 2.4

Database: MySQL

PHP version: 5.5 or 5.4.

ownCloud version: 8.1.3 for one instance, 8.2 for the other.

Storage backend: NFS and local disk.
### Client configuration

Client version: Version 2.0.2

Operating system: Ubuntu 14.04

OS language: English

Installation path of client: Wherever the .deb puts it.
### Comments

I recently set up a personal OwnCloud instance, and started using it as well as my workplaces instance. When my workplace account screen kept saying "Waiting for 1 other folder(s)" I had no idea what it meant. It took Googling to find https://github.com/owncloud/client/issues/3619 to figure it out.

Can I suggest the language be changed to something like:

```
Waiting for another OwnCloud account to finish syncing. Client does not support syncing multiple accounts at the same time.
```

Getting the language correct is important. I will eventually be telling users at work how to use multiple accounts. I know for a fact that if it confuses me, the person running our OwnCloud instance, it will confuse them.

**Comment by jancborchardt** [comment]
Yeah, good find. It should even be so explicit as to say which account it is, but still be a concise message so people will actually read it.

> Waiting for jerrac@examplecloud.com to finish syncing

or maybe

> Waiting to finish sync of jerrac@examplecloud.com

or (although that might be too short)

> Finishing sync of jerrac@examplecloud.com


What do you think @guruz @jerrac? Which wording is the best? I think the additional note that parallel syncing is not supported is not necessary, just pointing out a shortcoming that most people will not notice or care about as long as they know that the account _will_ sync once the other one is finished.

**Comment by jerrac** [comment]
I like the second message best.

Is the fact that parallel syncs are not supported documented anywhere other than github? So that a non-technical user could figure it out if they do end up noticing. 

I didn't see anything on this page: https://doc.owncloud.org/desktop/2.0/navigating.html#multi-account-support

**Comment by jancborchardt** [comment]
The second is my favorite as well. @guruz can you change it?

**Comment by ogoffart** [comment]
It's really hard to know from the UI which folder it is waiting on.  And it can be several folders.

How about 

> Waiting to finish sync of another folder sync connection
?

**Comment by phil-davis** [comment]
I had been thinking about the idea that I have now put in #4039 - as well as sorting out the exact text to put here, showing status by account will also help a user to quickly find the "other folder sync connection".

**Comment by jancborchardt** [comment]
@ogoffart well, it doesn’t seem to solve the original problem, which was that @jerrac was confused that the word »folder« was used when it was another account.

**Comment by jerrac** [comment]
> Waiting to finish sync of another folder sync connection

That isn't really different from the current message. 

If you can't get the actual account name, maybe:

> Waiting for your other account to finish operations.

The idea being that they've already clicked the "Add an Account" button, so they would know what is meant by "account".

**Comment by phil-davis** [comment]
It is possible to have a folder sync connection waiting for another folder sync connection in the same account. So the wording will need to work sensibly in all permutations. I will think...
![owncloud-multisyncinoneaccount](https://cloud.githubusercontent.com/assets/1535615/10838232/2f7e09e8-7eeb-11e5-9c56-2ea2dec10616.png)

**Comment by phil-davis** [comment]
Waiting for 1 other account folder sync(s) to finish...

Does that somehow combine the fact that there are potentially multiple accounts, and multiple folder sync connections in each account?

All ideas welcome :)

**Comment by jerrac** [comment]
Why not multiple messages?

> Waiting for your other account to finish syncing.

and 

> Waiting for another folder in this account to finish syncing.
